src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -171,6 +173,47 @@ const AdminDashboard = (props) => {
     { label: "Access Control" },
   ];
   
+  const QUERIES = {
+    "unpaidsessions": gql`query MyQuery {
```

Comment:
Be sure to rename `MyQuery` to something more descriptive (e.g. "GetUnpaidSessions")